### ISSUE TITLE
unable to reproduce your experimental

### ISSUE THREAD

**Issue by chouss911** [WillDreamer/Aurora#14]
We are unable to reproduce your experimental results on the MSRVTT QA dataset, with an accuracy rate of around 31. May I ask for the reason, or can you provide some checkpoints

**Comment by zhangbw17** [comment]
And I tried to reproduce the retrieval result using msrvtt, trained with command
 `python3 -m torch.distributed.run --nproc_per_node=8 --master_port 12333 train_video_retrieval.py --config ./configs/retrieval_msrvtt.yaml --output_dir /tmp`
and tested with this command
 `python3 -m torch.distributed.run --nproc_per_node=8 --master_port 12333 train_video_retrieval.py --config ./configs/retrieval_msrvtt.yaml --evaluate --pretrained path/to/ckpt --output_dir /tmp`
And the rank@1 is 45.0, which is much lower than the reported the result. may you provide the msrvtt checkpoints? Thansk.
